### PR TITLE
feat(AAiT-web_g3): Add teams page

### PR DESCRIPTION
**Pull request by surafel58**
i have added teams page
have responsive feature
have pagination feature
i used dummy data defined in data folder

here is the screenshot of the UI
image

image

image

mobile Screen UI
image

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/app/teams/page.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+"use client";
+import HeadLineSection from "@/components/team/HeadLineSection";
+import Pagination from "@/components/team/Pagination";
+import { teams } from "@/data/team/Teams";
+import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
+
+const Teams = () => {
```
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/app/teams/page.tsx:9]: - add React Fc

File: aait/web/group-3/starter_project_group3/components/header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
+// Import necessary components and modules
+"use client";
+import Image from "next/image";
+import NavBar from "./NavBar";
+import { useState } from "react";
+import NavBarMobile from "./NavBarMobile";
+
+// Define a functional component named Header
+const Header = () => {
```
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/components/header/Header.tsx:9]: - add React Fc

File: aait/web/group-3/starter_project_group3/app/globals.css
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/app/globals.css:1]: - Try to add React Fc for all

**Review by BrookW-83 — COMMENTED** [review]
- Try to add React FC to your component

File: aait/web/group-3/starter_project_group3/app/teams/page.tsx (hunk 1 above)
Comment (surafel58) [reply aait/web/group-3/starter_project_group3/app/teams/page.tsx:9]: Okay.

**Review by BrookW-83 — APPROVED** [review]

**Pull request merged**